catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/ProjectListFragment.java

```diff
@@ -87,6 +87,30 @@
 
 	private ProjectController projectController = new ProjectController();
 
+	private ProjectUnzipAndImportTask.ProjectUnzipAndImportListener projectUnzipAndImportListener =
+			new ProjectUnzipAndImportTask.ProjectUnzipAndImportListener() {
+				@Override
+				public void onImportFinished(boolean success) {
+						adapter.setItems(getItemList());
+						if (!success) {
+							ToastUtil.showError(getContext(), R.string.error_import_project);
+						}
+						setShowProgressBar(false);
+					}
```

Comment:
indentation is weird here :confused:. (compare it to the override below). strange that no static checks complain about it. Can you have a look at it please?